### DISCUSSION TITLE
How to sign tag with 'rake release' command?

### DISCUSSION THREAD

**Discussion by the-pulli** [ruby/rubygems#6202] [Q&A] [answered]
Is there any way to sign the added tag with the `rake release` command which gets then pushed to GH?

Thanks!

**Comment by simi** [comment]
Whole logic of git push tag is at gem_helper.rb.

https://github.com/rubygems/rubygems/blob/72fd3dd2096af16d797ad0cd8e0d2a8869e240b3/bundler/lib/bundler/gem_helper.rb#L132-L133

Do you have signing enabled by default for your git configuration?

**Reply by the-pulli** [reply]
Doh! Replied to my own question, not to your answer 😅
See below.

**Comment by the-pulli** [comment]
Yeah, I saw that while researching, thanks!

For commits, I've signing enabled by default. I searched the net and found out that for signing tags there is no global flag. So one has to append the -s flag. Is that still the case?

**Comment by simi** [comment, marked as answer]
Can't you set this flag to make git tag signed by default?

https://git-scm.com/docs/git-tag#Documentation/git-tag.txt--s
https://git-scm.com/docs/git-config#Documentation/git-config.txt-taggpgSign

**Reply by the-pulli** [reply]
That did the trick! Thank you very much.
